assets/js/serviceworker.js

```diff
@@ -0,0 +1,90 @@
+var CACHE_NAME = 'sass-guidelines-cache';
+// The files we want to cache
+var urlsToCache = [
```

Comment:
Yes. I was going to say something like this. I’ll see if I can come up with something.